server/src/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -4932,6 +4932,32 @@ public Domain getVlanDomain(long vlanId) {
         return _networkOfferingDao.search(sc, searchFilter);
     }
 
+     @Override
+     @DB
+     public boolean releaseDomainSpecificVirtualRanges(final long domainId) {
+        final List<DomainVlanMapVO> maps = _domainVlanMapDao.listDomainVlanMapsByDomain(domainId);
+        if (maps != null && !maps.isEmpty()) {
```

Comment:
@ustcweizhou For a List you can use CollectionUtils.isEmpty (https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html#isEmpty%28java.util.Collection%29)